torch/nn/modules/rnn.py

```diff
@@ -1028,16 +1028,17 @@ class LSTMCell(RNNCellBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
-    Examples::
+    Examples::    
 
-        >>> rnn = nn.LSTMCell(10, 20)
-        >>> input = torch.randn(3, 10)
-        >>> hx = torch.randn(3, 20)
-        >>> cx = torch.randn(3, 20)
+        >>> rnn = nn.LSTMCell(10, 20)        # (input_size, hidden_size)
+        >>> input = torch.randn(2, 3, 10)    # (time_steps, batch, input_size)
+        >>> hx = torch.randn(3, 20)          # (batch, hidden_size)
+        >>> cx = torch.randn(3, 20)          
         >>> output = []
-        >>> for i in range(6):
+        >>> for i in range(input.size()[0]):  
```

Comment:
And here

Reply:
done! Thank you for your patience :) 
please let me know if I should correct anything else